ui/components/ui/icon/icon.stories.js

```diff
@@ -22,7 +22,10 @@ import SendIcon from './send-icon.component';
 import Sign from './sign-icon.component';
 import SunCheck from './sun-check-icon.component';
 import Swap from './swap-icon-for-list.component';
-import IconCheck from './icon-check';
+import IconCaretLeft from './icon-caret-left';
+import IconCaretRight from './icon-caret-right';
+import IconCaretDown from './icon-caret-down';
+import IconCaretUp from './icon-caret-up';
```

Comment:
I don't think these files exist anymore.

Reply:
<img width="677" alt="image" src="https://user-images.githubusercontent.com/53999998/219548978-6fe8aff0-d76d-45be-8929-e0cc4bfef015.png">
 these components are being used in the code, and I did not add them in there. Should I go ahead and remove all the imports and their usage giving errors?